lib/swagger/formatter.ex

```diff
@@ -50,7 +50,8 @@ defmodule Xcribe.Swagger.Formatter do
             summary: "",
             responses: responses_object_from_request(request),
             parameters: parameter_objects_from_request(request),
-            security: security_requirement_object_by_request(request)
+            security: security_requirement_object_by_request(request),
+            tags: [request.resource]
```

Comment:
When we have a composed resource it's separated by an underscore (eg `user_posts`). To turn into tag would be better replace the underscore with a space. WDT?

Reply:
Done!